exotations/dynamics_solvers/exotica_quadrotor_dynamics_solver/src/quadrotor_dynamics_solver.cpp

```diff
@@ -108,4 +108,25 @@ Eigen::VectorXd QuadrotorDynamicsSolver::GetPosition(Eigen::VectorXdRefConst x_i
     return xyz_rpy;
 }
 
+Eigen::Tensor<double, 3> QuadrotorDynamicsSolver::fxx(const StateVector& x, const ControlVector& u)
```

Comment:
Add a TODO comment for the whole class or maybe even throw an 'NotYetImplemented' exception when used.

Reply:
Now throwing exceptions on `fx` and `fu`, which are not implemented. The second-order derivatives are 0 by default.